modules/node_modules/@ncigdc/containers/FileAggregations.js

```diff
@@ -96,58 +185,18 @@ export const FileAggregationsComponent = (props: TProps) => (
   );
 
 export const FileAggregationsQuery = {
+  initialVariables: _.mapValues(initialFileAggregationsVariables, (value, key) => {
+    const userSelectedFacetsFromStorage = JSON.parse(window.localStorage.getItem(storageKey)) || [];
```

Comment:
`Uncaught SyntaxError: Unexpected token u in JSON at position 0`
cuz `window.localStorage.getItem(storageKey)` is returning the string "undefined"